examples/annotation_import/tiled.ipynb

```diff
@@ -20,12 +20,12 @@
       },
```

Comment:
<p> </p><pre class="ql-syntax" spellcheck="false">from labelbox.schema.ontology import OntologyBuilder, Tool, Classification, Option
from labelbox import Client, LabelingFrontend, LabelImport, MALPredictionImport

</pre><p>These are all available in the top level namespace.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>